maskit/masks.py

```diff
@@ -34,10 +34,17 @@ class Mask(object):
     __slots__ = ("mask", "_parent")
 
     def __init__(
-        self, shape: Tuple[int, ...], parent: Optional["MaskedCircuit"] = None
+        self,
+        shape: Tuple[int, ...],
+        parent: Optional["MaskedCircuit"] = None,
+        mask: Optional[np.ndarray] = None,
```

Comment:
Please document these parameters in the docstring.

Reply:
Also applies to the other new parameters of other classes.

Reply:
Done. I also included the others O:)